match/lib/match/utils.rb

```diff
@@ -18,7 +18,7 @@ def self.fill_environment(params, uuid)
     end
 
     def self.environment_variable_name(params)
-      ["sigh", params[:app_identifier], params[:type]].join("_")
+      ["sigh", params[:app_identifier], params[:type], params[:platform]].join("_")
```

Comment:
Is there code in `sigh` that uses this environment variable that needs to be updated? If this is for external use, it would seem like a backwards incompatible change for users...